examples/invoices/json.yml

```diff
@@ -38,72 +34,71 @@ get-single: |
         "DueDateString": "2009-06-06T00:00:00",
         "Status": "AUTHORISED",
         "LineAmountTypes": "Exclusive",
-        "LineItems": {
-          "LineItem": {
+        "LineItems": [
+          {
             "Description": "Onsite project management ",
             "Quantity": "1.0000",
             "UnitAmount": "1800.00",
             "TaxType": "OUTPUT",
             "TaxAmount": "225.00",
             "LineAmount": "1800.00",
             "AccountCode": "200",
-            "Tracking": {
+            "Tracking": [
               "TrackingCategory": {
```

Comment:
the "TrackingCategory" name should be deleted so it becomes `Tracking: [ { TrackingCategoryID... } ]`